aptos-move/move-examples/wrapped_token/sources/wrapped_token.move

```diff
@@ -0,0 +1,110 @@
+/// This provides an example for how to wrap a Token V1 in a Token V2
```

Comment:
Can you add more info here (ChatGPT?) about what a wrapped TokenV2 means, why do so and the basic user flow?